#asr result message

string  text    #语音转换成的文本
# string  voice   #发音人